test/metabase/test/data/h2.clj

```diff
@@ -51,7 +50,7 @@
   (defmethod sql.tx/field-base-type->sql-type [:h2 base-type] [_ _] database-type))
 
 (defmethod tx/dbdef->connection-details :h2
-  [_driver context dbdef]
+  [_ context dbdef]
```

Comment:
Some of the stuff like this is just a general improvement and we don't need to revert it